versions/3.0.3.md

```diff
@@ -3331,6 +3333,36 @@ flows:
       read:pets: read your pets 
 ```
 
+###### OAuth Flows with Optional or Unspecified Scope
```

Comment:
Let's remove this, please. Adding more examples for edge cases just makes the spec longer and harder to read. This belongs in a tutorial.